holidayextras/holidayextras.github.com
# Atom feed for posts in the Testing category

#### What does this PR do? (please provide any background)
We already had an Atom feed (`feed.xml`) that supplied a feed of the 10 most recent blog posts across all categories. I've been asked to supply a feed of posts in the Testing category so that they can be syndicated by Ministry of Testing. I have therefore created a second Atom feed (`testing-feed.xml`) that does the job using `{% for post in site.categories.testing limit:10 %}`.

#### What tests does this PR have?
N/a

#### How can this be tested?
Look at the newly created `testing-feed.xml` in a text editor, or open it in your browser to check that it is parsed correctly.

#### What gif best describes how you feel about this work?

![](http://45.media.tumblr.com/tumblr_les8op0Bsn1qb7icgo1_500.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
